### PR TITLE
adding generate_a_HtmlFile_from_mdFile test

### PR DESCRIPTION
**Pull request by hphan9**
Hi Qiwen-Yu, I added 1 test to check if an HTML file will be generated in the expected destination if the user passes in an MD file.
Thank you :)

### CODE REVIEW

**Comment by Qiwen-Yu** [comment]
> Hi Qiwen-Yu, I added 1 test to check if an HTML file will be generated in the expected destination if the user passes in an MD file. Thank you :)

Thank you for your PR.

**Review by Qiwen-Yu — COMMENTED** [review]
The test did not pass. I think the reason is the destination `../TextHTMLPress/dist` does not exist on GitHub.

File: test.py [hunk 1]
```diff
@@ -36,3 +36,13 @@ def test_md() -> None:
...
+
+def test_generate_a_HtmlFile_from_mdFile()->None:
+    inp=Path("./tests/inputs/example1.md")
+    output = Path("../TextHTMLPress/dist")
+    expectedOutputFile= Path("../TextHTMLPress/dist/example1.html")
+    stylesheet = ""
+    lang = "fr"
+    g=Generator(inp,output,stylesheet,lang)
+    g.parse_markdown(inp)
+    assert expectedOutputFile.exists()
```
Comment (Qiwen-Yu) [inline test.py]: The above line might be changed something as """<!DOCTYPE html><html lang="en-CA"><head><meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" /><meta name="author" content="Somebody" /><title>Zen</title></head><body><h3>Somebody</h3><h3>2021-10-15</h3><h1>Zen</h1>
<h2>Section 1</h2>
<p>Simple is better than complex.</p>
<p>Although practicality beats purity.</p>
<h2>Section 2</h2>
<p>Need more practice.</p></body></html>"""

Comment (hphan9) [reply test.py]: It is so strange when I test it local and all tests pass

Comment (hphan9) [reply test.py]: I got it, I forgot to generate wrapper in the test

**Comment by Qiwen-Yu** [comment]
I will merge now, but I am wondering if we should remove the expectedOutputFile first such as `shutil.rmtree(expectedOutputFile)`, then use the `Generator`, and finally use `assert` to check.

**Pull request merged**